.ci/build-images.sh

```diff
@@ -61,9 +82,23 @@ function _pull_dev_dependencies
 ##
 function _main
 {
-    _build "arm"
-    _build "x86"
-    _build "dev"
+
+    # builds x86 and arm images based on manifest files
+    if [[ ! -z ${BUILD_TAG} ]]
+    then
+        GIT_MANIFEST_BRANCH=refs/tags/${BUILD_TAG}
+        _build "${DOCKERFILE_PATH}/arm" "arm" "--no-cache"
```

Comment:
I don't see the point of these two lines compare to the two ones after the end of if

Reply:
I forgot to remove the ones outside the if case